plugin/trino-influxdb/src/test/java/io/trino/plugin/influxdb/TestInfluxMetadata.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.influxdb;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import io.trino.spi.connector.ColumnMetadata;
+import io.trino.spi.connector.ConnectorTableMetadata;
+import io.trino.spi.connector.SchemaTableName;
+import io.trino.spi.connector.SchemaTablePrefix;
+import io.trino.spi.connector.TableColumnsMetadata;
+import io.trino.spi.connector.TableNotFoundException;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.Optional;
+
+import static io.trino.plugin.influxdb.InfluxConstant.ColumnName.TIME;
+import static io.trino.plugin.influxdb.InfluxDataTool.TEST_DATABASE;
+import static io.trino.plugin.influxdb.InfluxDataTool.TEST_DATABASE_ANOTHER;
+import static io.trino.plugin.influxdb.InfluxDataTool.TEST_MEASUREMENT_X;
+import static io.trino.plugin.influxdb.InfluxDataTool.TEST_MEASUREMENT_Y;
+import static io.trino.plugin.influxdb.InfluxDataTool.TEST_MEASUREMENT_Z;
+import static io.trino.spi.type.BigintType.BIGINT;
+import static io.trino.spi.type.BooleanType.BOOLEAN;
+import static io.trino.spi.type.DoubleType.DOUBLE;
+import static io.trino.spi.type.VarcharType.VARCHAR;
+import static io.trino.testing.TestingConnectorSession.SESSION;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertNull;
+
+public class TestInfluxMetadata
+{
```

Comment:
It would be nice if you can change this class to query based instead of direct `InfluxMetadata`.

Reply:
ok , I write these tests follow as `TestDefaultJdbcMetadata`.  i understand now that we best to test behavior using query as you said, so i removed those tests such as TestInfluxColumn、 TestInfluxTable ...
If i can remove this `TestInfluxMetadata` or keep it left ? please give me a suggestion.